### ISSUE TITLE
I just updated hyprland-git from AUR and it keeps crashing on startup

### ISSUE THREAD

**Issue by nnra6864** [hyprwm/Hyprland#8438]
### Already reported ? *

- [X] I have searched the existing open and closed issues.

### Regression?

Yes

### System Info and Version

I can't run hyprland, and therefore can't run this command.
I am using Arch, Zen, and all the git versions of hyprland related packages.
I also have an nvidia 2080ti, that previously worked just fine.

### Description

I genuinely have no idea, I just updated from a previously fine working system.
I tried renaming my actual config in order to use the default hyprland config, didn't help either.

### How to reproduce

Might be possibly to reproduce, it could be that I messed something up, but afaik, I just updated.
Literally just launch Hyprland from a tty, that's it.

### Crash reports, logs, images, videos

I couldn't get a log since Hyprland doesn't exactly crash, it keeps running in the tty but I get no gui.
I uploaded a short video on your discord server and made it print to std output so here's that:
[HyprlandLog.txt](https://github.com/user-attachments/files/17712245/HyprlandLog.txt)
https://discord.com/channels/961691461554950145/967398886530515067/1305769619545657354

**Comment by nnra6864** [comment]
Ok, well I managed to get into Hyprland by cloning the repo and building that.
It comes with it's own set of issues tho, the rendering is completely borked...
![image](https://github.com/user-attachments/assets/f3f97545-fcbf-48ae-a25f-665199fc74c6)

I narrowed it down to blur being enabled, no matter what settings you use, and will try to remove commits 1 by 1 to see if I can pinpoint the issue.
Do let me know if I am missing something really obvious here tho.

**Comment by fufexan** [comment]
`git bisect` uses binary search and is way faster at finding bad commits than removing them 1 by 1.

**Comment by vaxerski** [comment]
looks like you compiled with legacy_renderer which has blur broken

**Comment by nnra6864** [comment]
> looks like you compiled with legacy_renderer which has blur broken

Quite possibly, I simply did the following:
`git clone --recursive https://github.com/nnra6864/Hyprland && cd Hyprland && make`
And then ran Hyprland with the following command:
`./build/Hyprland -c ~/.config/hypr/hyprland.conf`
(I have to specify the config because I somehow managed to make Hyprland always use hyprlandd.conf no matter what I do or how I compile it :/)

I also found the following [solution](https://discord.com/channels/961691461554950145/1270460936301314048/1305471037881712650), which didn't work for me if I did it correctly.
I simply added `force_introspection = 2` to my config, which should already be a default value on nv gpus if I am not mistaken, so I tried 0 and 1 as well, and still the same.

As for the aur hyprland crashing, I'll try updating once again, could be fixed by now.

**Comment by vaxerski** [comment]
yes, that's what happens when you disregard any and all instructions on how to compile on the wiki

`make release` bud, not `make`

**Comment by nnra6864** [comment]
Welp, you nailed it, I indeed have disregarded any and all the instructions :man_facepalming: 
I didn't have any issues with just running make previously so I thought it was fine, guess I just got lucky.

**Comment by rabiddoughnuts** [comment]
> yes, that's what happens when you disregard any and all instructions on how to compile on the wiki
> 
> `make release` bud, not `make`

I am having the same issue, and am confused, cause you say the directions are 'make release' but I'm looking at the wiki rn, and they say 'make all'

**Comment by vaxerski** [comment]
`make all` = `make clear && make release`
